torch/onnx/symbolic_opset11.py

```diff
@@ -894,6 +894,21 @@ def embedding_bag(g,
     return loop.node().output(), None, None, None
 
 
+@parse_args('v', 'v', 'f', 'f')
+def embedding_renorm(g, weight, indices, max_norm, norm_type):
+    unique_indices = g.op("Unique", indices)
+    partial_weight = g.op("Gather", weight, unique_indices)
+    norm_type = "ReduceL2" if int(norm_type) == 2 else "ReduceL1"
+    partial_weight_norm = g.op(norm_type, partial_weight, axes_i=[1], keepdims_i=1)
+    partial_weight_norm_ = g.op("Add", partial_weight_norm, g.op("Constant", value_t=torch.tensor(1e-7)))
```

Comment:
add a link to the source for 1e-7. In general constants like this should have some explanation.

Reply:
added comments